app/javascript/dashboard/api/contactNotes.js

```diff
@@ -0,0 +1,9 @@
+import ApiClient from './ApiClient';
+
+class ContactNotes extends ApiClient {
+  constructor() {
+    super('contact_notes', { accountScoped: true });
+  }
+}
+
+export default new ContactNotes();
```

Comment:
Add specs for `contactNotes` API

Reply:
The api is not yet ready for contacts. I'll add once API is ready in another PR